Source/Foundation/bsfUtility/Utility/BsBitstream.h

```diff
@@ -9,8 +9,17 @@
 #include "Math/BsVector3.h"
 #include "Utility/BsBitwise.h"
 
+
 namespace bs 
 {
+	using QuantType = uint8_t;
+	// see https://stackoverflow.com/questions/52355223/why-cant-a-static-constexpr-member-variable-be-passed-to-a-function
+	// static constexpr variables before c++17 can't be defined within the class.
+	static constexpr uint32_t BYTES_PER_QUANT = sizeof(QuantType);
```

Comment:
They can be part of the class, they just need to be declared in the .cpp file as well, for certain cases (very compiler dependent so far). I'd prefer that option than moving it out of class.

Reply:
yea there just wasn't an associated cpp file so I wasn't sure if it was worth the effort